packages/iTwinUI-react/src/core/Dialog/DialogBackdrop.tsx

```diff
@@ -53,6 +54,7 @@ export const DialogBackdrop = React.forwardRef<
       isVisible={isVisible}
       ref={refs}
       onMouseDown={handleMouseDown}
+      style={{ pointerEvents: 'auto', ...style }}
```

Comment:
Since https://github.com/iTwin/iTwinUI/pull/739 is merged into the v1 branch and won't be available for a while, maybe add inline `position: fixed` to this.

Reply:
Added